backend/src/routes/v0/project/__init__.py

```
@@ -109,6 +110,40 @@ async def get_project(
+################################################################################
+### Update Project
+################################################################################
+@router.put(
+    "/{project_id}",
+    name="Update Project",
+    operation_id="ruska_update_project",
+    tags=["mcp"],
+)
+async def update_project(
+    project_id: str,
+    project: ProjectUpdate = Body(...),
```

Ruff (0.14.11)

Do not perform function call `Body` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)